tool/tctl/common/resource_command.go

```diff
@@ -642,6 +644,76 @@ func (rc *ResourceCommand) createNode(ctx context.Context, client auth.ClientI,
 	return trace.Wrap(err)
 }
 
+func (rc *ResourceCommand) createConnector(ctx context.Context, client auth.ClientI, raw services.UnknownResource) error {
```

Comment:
Why is this function responsible for two different resource kinds?

Reply:
A question for the history books, I suppose. I wouldn't want to rewrite the SAML connector logic as part of a PR that introduces a new command, so if we want to revisit this later we can. It hasn't been a problem to date so I don't think it's urgent.